bench/Autofac.Extensions.DependencyInjection.Bench/BenchmarkConfig.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Jobs;
+
+namespace Autofac.Extensions.DependencyInjection.Bench
+{
+    internal class BenchmarkConfig : ManualConfig
+    {
+        internal BenchmarkConfig()
+        {
+            Add(DefaultConfig.Instance);
+
+            var rootFolder = AppContext.BaseDirectory.Substring(0, AppContext.BaseDirectory.LastIndexOf("bin", StringComparison.OrdinalIgnoreCase));
+            var runFolder = DateTime.Now.ToString("u").Replace(' ', '_').Replace(':', '-');
+            ArtifactsPath = $"{rootFolder}\\BenchmarkDotNet.Artifacts\\{runFolder}";
```

Comment:
Use `Path.Join` so it doesn't bork on my Mac ;)

Reply:
Should probably update this in the main Autofac project where I took it from...

Reply:
Probably, but since main Autofac targets .NET 4.6 I can't build it on Mac. (Well, not easily, there are some weird things you can do with reference libraries and things, but you can't really execute it or test it. Would love to drop desktop .NET to allow for an easier build experience...)

Reply:
![gif](https://media.tenor.com/images/0fa0f72f2e14f9adb5cc4056ad007d33/tenor.gif)